src/main.c

```diff
@@ -236,9 +222,22 @@ __attribute__((section(".boot"))) int main(void)
 		BEGIN_TRY {
 			TRY {
 				io_seproxyhal_init();
+
+				#if defined(TARGET_NANOX)
+				// grab the current plane mode setting
+				// this line crashes the app
+				//G_io_app.plane_mode = os_setting_get(OS_SETTING_PLANEMODE, NULL, 0);
+				#endif
+
 				USB_power(0);
 				USB_power(1);
 				ui_idle();
+
+				#if defined(HAVE_BLE)
+				BLE_power(0, NULL);
+				BLE_power(1, "Nano X");
```

Comment:
Do we know what is this second parameter use for?

Reply:
I guess it should be the name under which the device is discoverable. But even if I tried changing it, my laptop kept showing the same name (the one set in Ledger device settings). It would be interesting asking ledger devs what should be the effect of this parameter then. Other apps have "Nano X" as well, but perhaps it would be nice to have there the app name, although I'm afraid that the name may be cached by the laptop/mobile and it may be actually misleading.